core/io/file_access_pack.cpp

```diff
@@ -36,11 +36,11 @@
 
 #define PACK_VERSION 1
 
-Error PackedData::add_pack(const String &p_path) {
+Error PackedData::add_pack(const String &p_path, bool p_overwrite_current) {
```

Comment:
I would use the argument name "p_replace_files", which I believe is a bit more descriptive.

Reply:
indeed :+1: